### PR TITLE
C_Project new window projectdatamodel

### PR DESCRIPTION
**Pull request by MHNV**
C_Project new window projectdatamodel
requirements: 
https://github.com/metasfresh/me03/issues/12458#issuecomment-1180082167

### CODE REVIEW

File: misc/services/camel/de-metas-camel-externalsystems/core/src/main/resources/application.properties [hunk 1]
```diff
@@ -114,7 +114,6 @@ metasfresh.prices-v2.api.uri = {{metasfresh.api.v2.baseurl}}/prices
 metasfresh.upsert-project-v2.camel.uri = {{metasfresh.api.v2.baseurl}}/project
 metasfresh.upsert-workorder-project-v2.camel.uri = {{metasfresh.api.v2.baseurl}}/project/workorder
 
-
```
Comment (metas-rc) [inline misc/services/camel/de-metas-camel-externalsystems/core/src/main/resources/application.properties:117]: Not an actual change. Can be reverted.

**Review by metas-rc — CHANGES_REQUESTED** [review]
Please, don't commit files where you only added an empty line.

File: backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Project.java [hunk 2]
```diff
@@ -1098,6 +1097,27 @@ public interface I_C_Project
...
 	String COLUMNNAME_SalesRep_ID = "SalesRep_ID";
 
+	/**
+	 * Set Specialist Consultant.
+	 *
+	 * <br>Type: String
+	 * <br>Mandatory: false
+	 * <br>Virtual Column: false
+	 */
+	void setSpecialist_Consultant (@Nullable java.lang.String Specialist_Consultant);
```
Comment (metas-rc) [inline backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Project.java:1107]: so the Specialist_Consultant is, in the end, metasfresh generic?

Comment (teosarca) [reply backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Project.java]: @MHNV cc @metas-rc  also why is string?!

**Pull request merged**